modules/ROOT/pages/creating-and-managing-a-cluster-manually.adoc

```diff
@@ -193,22 +211,17 @@ If set to `true`, do not set IP addresses in `mule.cluster.nodes`.
 |`mule.cluster.multicastgroup` |Multicast group IP address to use.
 |`mule.cluster.multicastport` |Multicast port number to use.
 | [[persistency-config]] `mule.cluster.jdbcstoreurl` | *Required only when storing persistent data*. +
-The JDBC URL for connection to the database
+The JDBC URL for connection to the database.
 |`mule.cluster.jdbcstoreusername` |*Required only when storing persistent data*. +
-Database username
+Database username.
 |`mule.cluster.jdbcstorepassword` |*Required only when storing persistent data*. +
-Database user password
+Database user password.
 |`mule.cluster.jdbcstoredriver` |*Required only when storing persistent data*. +
-JDBC Driver class name
+JDBC Driver class name.
 |`mule.cluster.jdbcstorequerystrategy` |*Required only when storing persistent data*. +
 SQL dialect for accessing the stored object data. +
-This property can take three different values: `mssql`, `mysql` and `postgresql`^<<anchor-1,1>>^.
-|`mule.cluster.jmxenabled`| *(Boolean)* Enable/disable monitoring
-| `mule.cluster.listenersenabled` | *(Boolean)* Enable/disable membership listener. Set to `true` if you want your nodes to get notified when a member enters or leaves the cluster.
+This property can take three different values: `mssql`, `mysql` and `postgresql`. +
+|`mule.cluster.jdbcstoretableNametransformerstrategy` | Allows you to define a custom transformer to modify table names. +
+|`mule.cluster.jmxenabled` | *(Boolean)* Enable/disable monitoring.
```

Comment:
what Boolean value is accepted? yes/no? 0/1? on/off? enable/disable? Please specify.

Reply:
We try to avoid bold for anything but UI labels and elements.

Reply:
I see below it's probably true/false. Pls specify here though. Thx!